src/main/kotlin/car_racing/Game.kt

```diff
@@ -0,0 +1,30 @@
+package car_racing
+
+class Game(
+    numCar: Int,
+    private val numMovement: Int,
+) {
+
+    var recordList: List<List<Int>> = listOf()
+        private set
+
+    private val cars: List<Car> = (1..numCar).map { Car() }
```

Comment:
```suggestion
    private val cars: List<Car> = List(numCar) { Car() }
```

개인적으로는 이 방법이 조금 더 보기 좋다고 생각해요. 리뷰이님의 생각은 어떠신가요?

Reply:
오! 훨씬 보기 좋은 것 같습니다  :)